css/index.css

```diff
@@ -1 +1,493 @@
+    	margin-bottom: 0px;
+    	line-height: 1em;
+    }
+
+    .contact {
+    	margin-top: 35px;
+    	padding-top: 0px;
+    	width: 92%;
+    	margin-left: auto;
+    	margin-right: auto;
+    }
+}
+
+@media screen and (max-width: 500px) {
+    .container {
+        width: 100%;
+    }
+
+    header {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        justify-content: center;
+    }
+
+    header nav {
+        display: flex;
+        flex-wrap: wrap;
+        width: 100%;
+        margin: 0 0 3em 0;
+    }
+
+    header nav a {
+        display: flex;
+        width: 100%;
+        justify-content: center;
+        align-items: center;
+        height: 6.1rem;
+        border-bottom: 1px solid black;
+        margin: 0 auto;
+        font-size: 2rem;
+    }
+
+    header .logo_container {
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 7.6rem;
+        order: -1;
+        border-bottom: 1px solid black;
+        margin: 0px;
+    }
+
+    header .logo_container .logo {
+    	height: 36px;
+    }
+
+    .cta {
+    	height: 376px;
+    	margin: 0 auto;
+    	width: 91%;
+    }
+
+    .cta .cta-text {
+        display: flex;
+        width: 100%;
+        text-align: center;
+        display: inline-block;
+        vertical-align: top;
+        font-size: 7.2rem;
+        text-align: center;
+        margin-top: 33px;
+        align-content: center;
+      }
+
+    .cta .cta-text button {
+        background: white;
+        vertical-align: top;
+        margin-top: 1rem;
+        padding: 2px 25px;
+        font-size: 16px;
+        border: 1px solid black;
+        cursor: pointer;
+        width: 170px;
+        height: 32px;
+    }
+
+    .cta img {
+        display: none;
+    }
+
+    .cta {
+        border-bottom: black solid 2px;
+        height: auto;
+        padding-bottom: 44px;
+    }
+
+    .main-content {
+        width: 97%;
+        border: none;
+        margin: 12px auto 0 auto;
+    }
+
+    .main-content h4 {
+        font-size: 1.6rem;
+    }
+
+    .main-content .top-content {
+        flex-wrap: wrap;
+        width: 95%;
+    }
+
+    .main-content .top-content .text-content {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+    }
+
+    .main-content .top-content .text-content p{
+        margin-top: 16px;
+        font-size: 1.6rem;
+        margin-bottom: 5px;
+    }
+
+    .main-content .top-content div:nth-child(2) {
+    	margin-top: 38px;
+    }
+
+    .main-content .bottom-content {
+        display: flex;
+        width: 95%;
+        flex-wrap: wrap;
+        margin-top: 28px;
+    }
+
+    .main-content .bottom-content .text-content {
+        width: 100%;
+        margin-bottom: 63px;
+        margin-top: 0;
+      }
+
+    .main-content .bottom-content .text-content:nth-child(3) {
+        margin-bottom: 0px;
```

Comment:
Remember that you can use `margin-bottom: 0;` as a shorthand over having the extra `px`.